.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  ignorePatterns: ["node_modules/*"],
```

Comment:
So which one works finally? This, `.eslintignore`, or in `package.json`?
Please pick only one

Reply:
I'm not sure because I removed all of them and it still works